config/packages/security.yaml

```diff
@@ -5,14 +5,40 @@ security:
     # https://symfony.com/doc/current/security.html#loading-the-user-the-user-provider
     providers:
         users_in_memory: { memory: null }
+        app_user_provider:
+            entity:
+                class: App\Entity\Contact
+                property: email
```

Comment:
Ook dit kan uitgecomment omdat het alleen gebruikt wordt door de form_login method.

Reply:
Je mist hier dan nog wel de saml-provider

Reply:
Is in een latere pr opgenomen.